cmd/pitr/collector/collector.go

```diff
@@ -97,10 +94,17 @@ func (c *Collector) newDB() error {
 	if err != nil {
 		return errors.Wrap(err, "get host")
 	}
-	c.db, err = pxc.NewPXC(host, c.pxcUser, c.pxcPass)
+
+	file, err := os.Open("/etc/mysql/mysql-users-secret/xtrabackup")
 	if err != nil {
-		return errors.Wrapf(err, "new manager with host %s", host)
+		return errors.Wrap(err, "open file")
+	}
+	var passBuf bytes.Buffer
+	passBuf.ReadFrom(file)
```

Comment:
error is not checked

Reply:
`ioutil.ReadAll` would look simpler

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/706/commits/4d042b7f9c38e77a138d72eb86e43609de35427b

Reply:
Do we want make code more clear?
We can use `ioutill.ReadFile`😅

Reply:
next time)